resources/views/layouts/main.blade.php

```diff
@@ -297,18 +297,32 @@ class="nav-link @if(Request::routeIs('admin.settings.*')) active @endif">
     <!-- Content Wrapper. Contains page content -->
 
     <div class="content-wrapper">
-
-        @if(!Auth::user()->hasVerifiedEmail())
+        @if(!Auth::user()->hasVerifiedEmail() && strtolower($force_email_verification) == 'true')
             @if(Auth::user()->created_at->diffInHours(now(), false) > 1)
                 <div class="alert alert-warning p-2 m-2">
-                    <h5><i class="icon fas fa-exclamation-circle"></i> Warning!</h5>
+                    <h5><i class="fas fa-envelope"></i> Warning!</h5>
                     You have not yet verified your email address <a class="text-primary"
                                                                     href="{{route('verification.send')}}">Click here to
                         resend
                         verification email</a> <br>
                     Please contact support If you didn't receive your verification email.
                 </div>
+            @else
+                <div class="alert alert-warning p-2 m-2">
+                    <h5><i class="fas fa-envelope"></i> Warning!</h5>
+                    Verification link has been sent to your email.<br>
+                    Please contact support If you didn't receive your verification email.
+                </div>
             @endif
+            
+        @endif
+        @if(is_null(Auth::user()->discordUser))
```

Comment:
this if statement is not conditional, not all hosts see a need in requiring discord verification